spec/feature/unauthenticated_guest_spec.rb

```diff
@@ -184,24 +182,94 @@
                 id: 1,
                 display_name: "joe")
     Listing.create(title: "Rainy day house",
-                  description: "house in the rain",
-                  category_id: 1,
-                  max_guests: 4,
-                  nightly_rate: 10000,
-                  address1: "124 Elm St",
-                  address2: nil,
-                  city: "Seattle",
-                  state: "WA",
-                  zip: 98106,
-                  shared_bathroom: true,
-                  user_id: 1)
+                   description: "house in the rain",
+                   category_id: 1,
+                   max_guests: 4,
+                   nightly_rate: 10000,
+                   address1: "124 Elm St",
+                   address2: nil,
+                   city: "Seattle",
+                   state: "WA",
+                   zip: 98106,
+                   shared_bathroom: true,
+                   user_id: 1)
     visit listings_path
-    find(:css, "#nightly_rate_15000[value='15000']").set(true)
-    click_link_or_button "Filter by price"
+    select('<$150', :from => 'nightly_rate')
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.